config/params-console-runner.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+return [
+    // Environment mode for application console runner.
+    'env' => null,
```

Comment:
```suggestion
    'environment' => null,
```

I think better full name for parameter.

Reply:
done.